scripts/create-fast-machine.sh

```diff
@@ -41,6 +41,8 @@ clever service link-addon c1-deployment-config --alias "$APP_NAME"
 clever service link-addon c1-imports-config --alias "$APP_NAME"
 clever service link-addon c1-prod-database-encrypted  --alias "$APP_NAME"
 
+# Ensure we've got the last version of master_clever
+git fetch
 clever deploy --alias "$APP_NAME" --branch master_clever --force
```

Comment:
Je pense qu'il manque le `origin/`:
```suggestion
clever deploy --alias "$APP_NAME" --branch origin/master_clever --force
```
(sinon, cela fait comme avant)